dracut/30ignition/ignition-fetch-offline.service

```diff
@@ -0,0 +1,28 @@
+[Unit]
+Description=Ignition (fetch-offline)
```

Comment:
Hmm, that would make it unlike all the other stages. I can imagine reworking them all though so that it's `Description=Ignition $stagename ($short_description)`. Does that work?

Reply:
IMO the other stages have clearer names.  The `$short_description` approach might be too verbose; I'm okay with it, or okay leaving as is.

Reply:
I'll leave it as is for now. We can always tweak descriptions in the future.